src/components/datagrid/DataGrid.js

```diff
@@ -60,6 +60,12 @@ export class DataGridComponent extends FormioComponents {
     if (!this.shouldDisable) {
       let th = this.ce('th');
       tr.appendChild(th);
+
+      // add the "+" - "add another" button
+      if (this.component.addAnotherPosition === "top" || this.component.addAnotherPosition === "both") {
```

Comment:
You still need a check here for ```!this.shouldDisable```

Reply:
Why do I still need to check !this.shouldDisable, the if statement is embedded inside the  check on line 60?

Reply:
didn't see that, so all good.